### PR TITLE
decouple the app and update path from the github repo hosting the files

### PR DESCRIPTION
**Pull request by joesis**
To have the ability to host update files in any repo without changing the URL from the client side

### CODE REVIEW

File: bin/entrypoint.sh [hunk 1]
```diff
@@ -7,14 +7,14 @@ if [ "$RUN_MODE" = "mock" ]; then
...
-    -repos getlantern/lantern
+    -repos lantern:getlantern/lantern
 else
   /bin/autoupdate-server \
     -k /keys/private.key \
     -l 0.0.0.0:9999 \
     -p https://update.getlantern.org/ \
     -o getlantern \
-    -r 0.3 \
+    -r 1000 \
```
Comment (joesis) [inline bin/entrypoint.sh:17]: Drive-by. Replace the rollout rate with rate limit. It's long been planed but I could not find the ticket.

**Comment by joesis** [comment]
To validate this change I did deploy this into production, watch update requests being processed, and verified that a newly built Lantern binary with version 5.0.0 gets updated to 6.0.10.

**Comment by myleshorton** [comment]
LGTM! I guess we'll just add the beam repo in as soon as we auto update.
